validator/nodejs/package.json

```diff
@@ -24,6 +24,6 @@
     "jasmine": "2.3.2"
   },
   "scripts": {
-    "postinstall": "/bin/sh -c \"exit 0\" 2> NUL && rm NUL || node postinstall-windows.js"
+    "postinstall": "node postinstall-windows.js"
```

Comment:
I think on Debian based systems (includes Ubuntu and LinuxMint) this will fail because the Node.js binary is called nodejs, not node. That's why this /bin/sh trickery exists...